training/hello_world_training.py

```diff
@@ -0,0 +1,85 @@
+from pathlib import Path
+from dataclasses import dataclass, field
+from math import pi
+
+from rlbot.utils.game_state_util import GameState, BoostState, BallState, CarState, Physics, Vector3, Rotator
+from rlbot.matchconfig.match_config import MatchConfig, PlayerConfig, Team
+from rlbottraining.common_exercises.common_base_exercises import StrikerExercise
+from rlbottraining.rng import SeededRandomNumberGenerator
+from rlbottraining.match_configs import make_empty_match_config
+from rlbottraining.grading.grader import Grader
+from rlbottraining.training_exercise import TrainingExercise
+
+import training_util
+from drive_to_ball_grader import DriveToBallGrader
+
+def make_match_config_with_my_bot() -> MatchConfig:
```

Comment:
Your youtube does a good job explaining the point of this, might want a comment linking to it here.

Reply:
I think whenever we import any python bot, we should look for requirements.txt to install.